package.json

```diff
@@ -35,6 +35,7 @@
     "lodash": "4.6.1",
     "mkdirp": "0.5.1",
     "moment": "2.12.0",
+    "path-is-inside": "1.0.1",
```

Comment:
Let's not bring in modules for stuff that can be done in one line.

I'm thinking we should only be allowing `a-zA-Z0-9_-` in usernames.